warehouse/schema_test.go

```diff
@@ -272,440 +276,170 @@ func TestHandleSchemaChange(t *testing.T) {
 	}
```

Comment:
Commenting here but my question is about line 274 meaning `require.Equal(t, convError, tc.convError)`.

Wouldn't it be semantically more correct to use `require.ErrorIs` so that the code will keep working if the error gets wrapped as well?